andi/andi.py

```diff
@@ -341,11 +379,41 @@ def _plan(class_or_func: Callable, *,
     return plan, flatten_errors
 
 
-def _select_type(types, is_injectable, externally_provided):
-    """ Choose the first type that can be provided. None otherwise. """
+def _select_type(types,
+                 is_injectable,
+                 externally_provided,
+                 overrides: Callable,
+                 recursive_overrides: bool
+                 ) -> Tuple[Optional[Callable], Callable]:
+    """
+    Choose the first type that can be provided. None otherwise. Also return
+    a boolean to keep track of overriding status.
```

Comment:
Type signature suggests that a callable is returned, not a boolean.

Reply:
It is not a boolean, the docstring was wrong. Updated here: https://github.com/scrapinghub/andi/pull/19/commits/0e9b2a9b44db067e9e17e18437be0b06964dae2b